dist/tools/ethos/start_network.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+trap "cleanup" EXIT #SIGINT SIGTERM
+
+create_tap() {
+    ip tuntap add ${TAP} mode tap user ${USER}
+    sysctl -w net.ipv6.conf.${TAP}.forwarding=1
```

Comment:
Is globally enabling forwarding a better option? that would disable SLAAC in the upstream interface, unless forwarding there is set to "2"...